File: 15792.cpp
#include <iostream>
using namespace std;

int main() {
	int A, B; cin >> A >> B;

	cout << A / B << '.';
	int tmp = A % B;
	for (int i = 0; i < 1000; i++) {
		tmp *= 10;
		int a = tmp / B;
		int b = tmp % B;
		cout << a;

		if (b == 0) break;
		else tmp = b;
	}
	cout << '\n';
}

File: 15792.py
A, B = map(int, input().split())

print(A // B, end='.')
tmp = A % B
for i in range(1000):
    tmp *= 10
    a = tmp // B
    b = tmp % B
    print(a, end='')

    if b == 0: break
    else: tmp = b
print()
